src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X509ChainPolicy.cs

```diff
@@ -30,6 +31,16 @@ public X509ChainPolicy()
         /// </value>
         public bool DisableCertificateDownloads { get; set; }
 
+        /// <summary>
+        ///   Gets or sets a value that indicates if VerificationTime was set and should be used or if it defaults to DateTime.Now from object creation.
+        /// </summary>
+        /// <value>
+        ///   <see langword="false" /> if VerificationTime was explicitly set and shall be used
+        ///   otherwise, <see langword="false" />.
+        ///   The default is <see langword="true" />.
```

Comment:
```suggestion
        ///   <see langword="true" /> to ignore <see cref="VerificationTime"/> and use the current system time; otherwise <see langword="false"/>.
        ///   The default is <see langword="true" />.
```